cypress/objects/LoginPage.js

```diff
@@ -0,0 +1,66 @@
+class LoginPage {
+    visit() {
+        cy.visit('/');
+    }
+
+    getUserValidationIndicator() {
+        return cy.get('.Select.not-valid');
+    }
+    getSubmitButton() {
+        return cy.get('[type="submit"]');
+    }
+
+    getSpecificUserFromDropDown(userName) {
+        return cy.get(`[aria-label="${userName}"]`)
+    }
+
+    getLoginFormButton() {
+        return cy.get('[id="loginForm.userId"]');
+    }
+
+    getTimesheets() {
```

Comment:
This locator is not in login page, but in timelogging page, it should be another file